src/client/Logo.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Logo = () => (
+  <div className="logo home-content__logo">
+    <span>{'{az}'}</span>
```

Comment:
Yeah, I just really wanted to abstract away the logo... Haha looking back, not the best idea